modules/ppcp-button/src/Assets/SmartButton.php

```diff
@@ -1070,4 +1074,30 @@ private function mini_cart_button_renderer_hook(): string {
 	private function single_product_renderer_hook(): string {
 		return (string) apply_filters( 'woocommerce_paypal_payments_single_product_renderer_hook', 'woocommerce_single_product_summary' );
 	}
+
+	/**
+	 * Check if cart product price total is 0.
+	 *
+	 * @return bool true if is 0, otherwise false.
+	 */
+	protected function is_cart_price_total_zero(): bool {
+		$cart_total = WC()->cart->get_cart_contents_total();
+		return ! ( $cart_total > 0 );
```

Comment:
Why not just `== 0` or `<= 0`? :)

Reply:
efca8ba27dde17fea3dc027f732bb2ac5306e635 done.